kit/wifi.md

```diff
@@ -46,7 +44,7 @@ The stop button that appears after starting your code is currently not functiona
 
 ### Setting up the Robot's Environment
 
-![The robot interface's set up screen]({{ site.baseurl }}/images/content/kit/tablet/set-up-screen.png)
+![The robot interface's set up screen]({{ site.baseurl }}/images/content/kit/remote-interfaceg/set-up-screen.png)
```

Comment:
Looks like there's a stray `g` in this path

Reply:
0617320